iiwa_tools/CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ add_service_files(
   GetJacobian.srv
   GetJacobians.srv
   GetGravity.srv
+  GetInertiaMatrix.srv
```

Comment:
I might be picky, but I think I prefer the term `MassMatrix` instead of `InertiaMatrix`. Isn't it the mass matrix that you are returning?

Reply:
I debated whether to do mass or inertia. I typically used mass matrix, but saw in the RBDyn library [here](https://github.com/costashatz/RBDyn/blob/master/src/RBDyn/FD.h#L58) it referenced as inertia matrix, so I attempted to comply with that language. If mass matrix is preferred I can change that.

On that note, would changing `Inertia` to `mass` be preferable for methods and member variables as well? I.e. `get_mass_matrix` and `_mass_service_name`?

Reply:
changed all instances to `mass` over `inertia`